### ISSUE TITLE
editing tasks from cloned challenge sometimes edits original challenge tasks

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#11185]
_Ceran, Hamster Juggler (@Ceran, aabe567b-aa55-42c1-b612-b7699a7ba9b7) reports:_

I've noticed this several times before, and it just happened again: Sometimes if I clone a challenge, and change the tasks of the new challenge, it will instead change the task in the mother challenge (the one I cloned from) and the task of the daughter challenge that I actually edited will revert to as it was before. [This](https://habitica.com/challenges/d5fed61a-5a3f-446a-b5e1-f47de635ba30) is the cloned challenge where I was trying to edit the to-do. [This](https://habitica.com/challenges/8c42020d-57d4-4399-a748-cfb8c89a7814) is the "mother" challenge that I cloned from. I am absolutely sure that I changed the task in the "daughter" challenge. But when I returned to the previous page in my browser, because I had noticed that the date in the to-do of the "mother" challenge had been wrong for that one anyway and wanted to adjust that too (as you can still see in the "daughter" challenge"), I then saw that it adjustment I had just made to the "daughter" challenge. And when I checked back with that one, indeed it had reverted back to its pre-editting state. I was going to award the "mother" challenge now, but I will leave it up a bit longer so that you can check it out.

**Comment by majestic-owl448** [comment]
I had this happen if you don't wait enough for the new challenge page to appear, so you are still on the parent challenge and could be editing those, but when you save the edit you are on the child challenge and the edit doesn't show, but if you go to check the parent challenge the edit is there

I don't know if this is what happens here, just wanted to bring this up as a possibility

**Comment by gautamz07** [comment]
@Alys is there anyway i can set gem to 1 locally so i can test this issue out ? As it says "need minimum 1 gen to create a challenge".

**Comment by Alys** [comment]
You can give yourself gems using the debug menu in the footer of the local install website but you can also change the challenge's location to any guild or party to make it free. It's only challenges that aren't in a guild/party that need a gem prize (so-called "public" challenges, although any challenge in a public guild can be free).

**Comment by Xaz16** [comment]
Hi folks!
@Alys can I take this task? As I understand @gautamz07 is not working on it anymore...

**Comment by Alys** [comment]
@gautamz07 How are you going with this? Would you like to keep working on it? If you ended up being too busy to be able to do it, just let us know - we understand what that's like. :)

**Comment by SabreCat** [comment]
Opening this up for some incoming volunteers to dig into, but @gautamz07 / @Xaz16 if you're planning to tackle it within the next week or so, speak up!

**Comment by Xaz16** [comment]
@SabreCat yeah, I will, but after closing some of PRs and issues, I really want to believe that I will work on it in week or so

**Comment by jbusa22** [comment]
Is this still open? I just attempted to recreate the error with no luck, maybe it has been fixed?
